graylog2-server/src/main/java/org/graylog/plugins/pipelineprocessor/functions/strings/KeyValue.java

```diff
@@ -110,6 +110,10 @@ public FunctionDescriptor<Map<String, String>> descriptor() {
                         trimValueCharactersParam
                 )
                 .description("Extracts key/value pairs from a string")
+                .ruleBuilderEnabled()
+                .ruleBuilderName("Convert key/value to map")
+                .ruleBuilderTitle("Convert key/value string '${value}' to map")
+                .ruleBuilderFunctionGroup(RuleBuilderFunctionGroup.CONVERSION)
```

Comment:
For consistency, can this function be renamed `KeyValueToMapConversion`, so it matches the naming convention of all other conversion functions?
It's not very intuitive that we have a special conversion function for String to Map; and another for Collection / JSON. Particularly since `MapConversion` is described very generically as: _Converts a map-like value into a map_. We don't want to break existing rules; but could we at least have `MapConversion` call `KeyValue` for String parameters? And perhaps deprecate `KeyValue`.

Reply:
Added this as https://github.com/Graylog2/graylog2-server/issues/16600